declarations/Coffee Meets Bagel.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Coffee Meets Bagel",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://coffeemeetsbagel.com/privacy-terms/",
+      "select": "#av_section_2"

```

Comment:
Use `startBefore` and `endBefore` selectors to select the content of the terms of service and privacy policy, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/8c7a4fe010546473f1f5aad24e48eebfe0d0de5d